Makefile

```
@@ -0,0 +1,22 @@
+.PHONY: docs install-pkgsite help
+
+PORT ?= 3030
+URL  := http://127.0.0.1:$(PORT)
+
+help:
+	@echo "Targets:"
+	@echo "  docs             Launch pkgsite on port $(PORT)"
+	@echo "  install-pkgsite  Install pkgsite"
+
+docs:
```

checkmake (0.3.2)

[warning] Target body for "docs" exceeds allowed length of 5 lines (8).

(maxbodylength)